pkg/controller/workload/workload_event.go

```diff
@@ -58,6 +61,10 @@ type Endpoint struct {
 	portList    []*workloadapi.Port
 }
 
+func SendData(uid string, hostName string, op string) {
+	SecurityDataChannel <- [3]string{uid, hostName, op}
```

Comment:
maybe construct a data structure to hold `uid`, `hostName` and `op` instead of using [3]string